tests/core/Wyam.Configuration.Tests/ConfigScript/CacheManagerFixture.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using NSubstitute;
+
+using NUnit.Framework;
+
+using Ploeh.AutoFixture;
+
+using Wyam.Common.Execution;
+using Wyam.Common.IO;
+using Wyam.Configuration.ConfigScript;
+using Wyam.Testing;
+using Wyam.Testing.IO;
+
+namespace Wyam.Configuration.Tests.ConfigScript
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.Self | ParallelScope.Children)]
+    public class CacheManagerFixture : BaseFixture
```

Comment:
Thanks for adding tests! How hard would it be to use test implementations instead of generated mocks? If the answer is "hard", then leave the tests as-is. Otherwise, I've been leaning towards full test object interface implementations over mocks recently.

Reply:
Yeah, I was considering both, but I was having trouble figuring out how to perform cleanup/setup in isolation (file system setup/teardown) and I didn't want the tests to take 4 seconds to run.

NCrunch might be the other reason (it's not happy with me right now). 😹